reactor-core/src/main/java/reactor/core/scheduler/SingleScheduler.java

```diff
@@ -133,6 +136,25 @@ public Disposable schedulePeriodically(Runnable task,
 				unit);
 	}
 
+	@Override
+	public String toString() {
+		StringBuilder ts = new StringBuilder(Schedulers.SINGLE)
+				.append('(');
+		if (factory instanceof Supplier) {
+			ts.append('\"').append(((Supplier) factory).get()).append('\"');
```

Comment:
Minor risky get() call

Reply:
ah I had that fixed to checking for instance of our own ThreadFactory specifically, must have been overridden in the rebase :(